### PR TITLE
PE-319 add alert content type

### PR DESCRIPTION
**Pull request by radmosley**

### CODE REVIEW

File: web/themes/custom/westy/src/components/icon/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Icon
+
+## Props
+
+- `name`: string - Name of icon, must be one from the "All Icons" list.
+    - Is the filename of an svg in the `web/themes/custom/westy/src/icon/svgs/` folder
```
Comment (joshuami) [inline web/themes/custom/westy/src/components/icon/readme.md:6]: This is not at all accurate anymore. We are just using FA for the icons. Please remove this readme or make it accurate.

File: web/themes/custom/westy/src/scss/components/_alerts.scss [hunk 2]
```diff
@@ -18,3 +18,17 @@ header.page__header {
...
+
+.global-alert-content {
+  p:last-child {
+    margin-bottom: 0;
+}
+  a {
+    color: $light;
+    text-decoration: underline;
+  }
+  .img {
```
Comment (joshuami) [inline web/themes/custom/westy/src/scss/components/_alerts.scss:30]: I don't think you need the .img as we don't have an SVG with that class outputting anywhere.

File: web/themes/custom/westy/templates/block/block--views-block--alert-block-alert.html.twig [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{#
+/**
+ * @file
+ * Template for a block.
```
Comment (joshuami) [inline web/themes/custom/westy/templates/block/block--views-block--alert-block-alert.html.twig:4]: Please add a note in the comments that this template is to override the utility class applied via block--views-block.html.twig.

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+{# TWIG #}
+<div class="{{ classes|join(' ')|trim }}" data-nid="{{ node.id }}" {% if node.changed %} data-changed="{{ node.changed.value }}" {% endif %}>
+	<div class="container">
+		<div class="d-flex flex-row p-3">
+			{{ alertIcon }}
+			<div>
+				<div class="fw-bold">{{ label }}</div>
+				<div class="global-alert-content">{{ content }}</div>
+				<span class="small">
+					Last updated:
```
Comment (joshuami) [inline web/themes/custom/westy/templates/content/node--alert--featured.html.twig:32]: Not a huge deal as we won't be translating Employees—at least not to my knowledge—but remember when you have text in a twig template escape it with a translation filter.

**Review by joshuami — CHANGES_REQUESTED** [review]
A couple small changes just to prevent future confusion, but otherwise looks good.

File: web/themes/custom/westy/src/scss/components/_alerts.scss [hunk 5]
```diff
@@ -18,3 +18,14 @@ header.page__header {
     }
   }
 }
+
+.global-alert-content {
+  p:last-child {
+    margin-bottom: 0;
+}
```
Comment (RichardDavies) [inline web/themes/custom/westy/src/scss/components/_alerts.scss:25]: Minor nit, but the indentation is not correct for this bracket.

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig [hunk 6]
```diff
@@ -0,0 +1,38 @@
+{# Set Alert Type #}
+{% if node.field_severity.value == 20 %}
+	{% set alertType = 'warning' %}
```
Comment (RichardDavies) [inline web/themes/custom/westy/templates/content/node--alert--featured.html.twig:3]: Tab spacing/indentation seems to be all over the place in this file... might as well correct it with Josh's other requested changes.

here's it's about a 8 space tab.

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+	{% set TypeName = 'warning' %}
+{% else %}
+	{% set alertType = 'info' %}
+	{% set TypeName = 'information' %}
+{% endif %}
+
+{% set alertIcon %}
+{% include "@westy/icon/icon.twig" with {
+    name: TypeName,
```
Comment (RichardDavies) [inline web/themes/custom/westy/templates/content/node--alert--featured.html.twig:12]: Here tab size is 1-3 spaces.

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    size: 'lg',
+  } only %}
+{% endset %}
+
+{# CLASSES #}
+{% set classes= [
+	  'bg-' ~ alertType,
```
Comment (RichardDavies) [inline web/themes/custom/westy/templates/content/node--alert--featured.html.twig:19]: Here tab size is again not consistent.

**Review by RichardDavies — CHANGES_REQUESTED** [review]
Noticed a few minor formatting issues. I'd normally let them slide, but since Josh is requesting some other changes, might as well address these too.

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+<div class="{{ classes|join(' ')|trim }}" data-nid="{{ node.id }}" {% if node.changed %} data-changed="{{ node.changed.value }}" {% endif %}>
+    <div class="container">
+        <div class="d-flex flex-row p-3">
+            {{ alertIcon }}
+            <div>
+                <div class="fw-bold">{{ label }}</div>
+                <div class="global-alert-content">{{ content }}</div>
+                <span class="small">
+                    Last updated:
+                    {{ node.changed.value|date('g:i a F j, Y') }}
```
Comment (joshuami) [inline web/themes/custom/westy/templates/content/node--alert--featured.html.twig:33]: Any particular reason for this format? We usually start with Month Day, Year and end with time on every other date format we have in use.

**Review by joshuami — CHANGES_REQUESTED** [review]

File: web/themes/custom/westy/templates/content/node--alert--featured.html.twig (hunk 9 above)
Comment (radmosley) [reply web/themes/custom/westy/templates/content/node--alert--featured.html.twig:33]: Its mirroring what I see on Portland.gov's Alert

Comment (joshuami) [reply web/themes/custom/westy/templates/content/node--alert--featured.html.twig:33]: Ah, that makes sense. And... we should update the Portland.gov alert to use a consistent date format. Please go ahead and match it to the format we have for created and updated on pages. I'll write up a story to address Portland.gov.

File: web/sites/default/config/views.view.admin_alerts.yml [hunk 10]
```diff
@@ -0,0 +1,834 @@
...
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url.query_args
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  page_1:
+    id: page_1
```
Comment (RichardDavies) [inline web/sites/default/config/views.view.admin_alerts.yml:170]: Can you change this machine name to something more descriptive? We've tried to standardize as a team on using the pattern `displaytype_name` so something like `page_alerts` would work.

File: web/sites/default/config/node.type.alert.yml [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    scroll_to_offset: '0'
+    highlight_on_scroll: 1
+    highlight_offset: '100'
+    sticky: 0
+    sticky_offset: '0'
+    sticky_stop: ''
+    sticky_stop_padding: '0'
+name: Alert
+type: alert
+description: 'An alert can be used to convey important information that all site users need to read. Sitewide alerts should appear prominently in a banner across the top of all sites until such time as the alerts are unpublished via manual or scheduled interaction. A user''s may dismiss an alert.'
```
Comment (RichardDavies) [inline web/sites/default/config/node.type.alert.yml:47]: There's a typo in this description. The last sentence should read "A user may..."

**Review by RichardDavies — COMMENTED** [review]
Found a couple of other minor items... Neither is critical enough to prevent merging, but I thought I'd see if you want to fix them first before I merge or not.

File: web/sites/default/config/views.view.admin_alerts.yml (hunk 10 above)
Comment (radmosley) [reply web/sites/default/config/views.view.admin_alerts.yml]: updated to page_admin_alerts.

File: web/sites/default/config/node.type.alert.yml (hunk 11 above)
Comment (radmosley) [reply web/sites/default/config/node.type.alert.yml]: fixed the typo

**Review by RichardDavies — APPROVED** [review]

**Pull request merged**